content/millennium/dstu2/scheduling/slot.md

```diff
@@ -0,0 +1,131 @@
+* [End](http://hl7.org/fhir/DSTU2/slot-definitions.html#Slot.end){:target="_blank"}
+* [Free-Busy Type](http://hl7.org/fhir/DSTU2/slot-definitions.html#Slot.freeBusyType){:target="_blank"}
+* [Scheduling Location Extension](#extensions)
+
+## Terminology Bindings
+
+<%= terminology_table(:slot, :dstu2) %>
+
+## Extensions
+
+* [Scheduling Location]
+
+### Custom Extensions
+
+All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/dstu2/StructureDefinition/{id}`
+
+ID                         | Value\[x] Type         | Description
+---------------------------|------------------------|-----------------------------------------------------------------------------------------------
+`scheduling-location`      | [`Reference`]          | Reference to the location corresponding to this schedule, where the appointment can be booked.
+
+## Search
+
+Search for Schedules that meet supplied query parameters:
+
+    GET /Slot?:parameters
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: true, system: true) %>
+
+### Parameters
+
+ Name             | Required?                                                        | Type           | Description
+------------------|------------------------------------------------------------------|----------------|------------------------------------------------------------------------------------
+`_id`             | This, or `slot-type`                                             | [`token`]      | The logical resource id associated with the resource.
+`slot-type`       | Yes when using `schedule.actor` and\or `-location`, or `id`      | [`token`]      | The type of appointments that can be booked into this slot. Example: `12345`
+`schedule.actor`  | This and `slot-type`, or `id`                                    | [`reference`]  | A single or comma separated list of Practitioner references. Example: `Practitioner/12345`
+`-location`       | This and `slot-type`, or `id`                                    | [`reference`]  | A single or comma separated list of Location references. Example: `12345`
+`date`            | Yes when using `slot-type`                                       | [`date`]       | The Slot date-time. Example: `2016`
+[`_count`]        | No                                                               | [`number`]     | The maximum number of results to be returned per page. Defaults to `50`.
+
+Notes:   
+
+- The `slot-type`, `schedule.actor`, and `-location` parameters may be included only once.
+  For example, `-location=1234,9876` is supported but `-location=1234&-location=9876`
+- `slot-type` is a required search parameter and must be specified along with `schedule.actor` and\or `-location`
+  For example, `type=1234&-location=9876` and `type=1234&-location=95671&schedule.actor=9876` are supported but `-location=9876` and `-location=95671&schedule.actor=9876` are not.
```

Comment:
There are a few examples here with the old "type" parameter instead of "slot-type"

Reply:
Addressed in https://github.com/cerner/fhir.cerner.com/pull/153/commits/5e9098c298c1764c1966b824fc93c33896719b31